src/api/tests/test_user_course.py

```diff
@@ -0,0 +1,40 @@
+from .util import Client
+
+TEST_USER = { 'email': 'test@email.com',
+              'password': '123456' }
+
+TEST_USER_COURSE = {'name': 'ADMN-1824',
+                    'semester': 'SUMMER 2020',
+                    'cid': '-1'
+}
+
+'''
+Test this api endpoint/file only with the following command line:
+pytest -s tests/test_user_course.py
+'''
+def test_user_course_post_success(post_user, client: Client):
+    '''
+    Test user course post by comparing it to user get course
+    '''
+    sess = client.post("/api/session", json=TEST_USER)
+    assert sess.status_code == 200
+    r = client.post("/api/user/course", json=TEST_USER_COURSE)
+    assert r.status_code == 200
+    g = client.get("/api/user/course", json=TEST_USER_COURSE)
+    get_data = g.json()[0]
```

Comment:
How is this assuring that your course is the first course in the results?

Reply:
On second thought, this is probably fine, other test cases also rely on a fresh database so you can leave it as this.